### ISSUE TITLE
Address the errors in the OAR task pool for H. Hadley

### ISSUE THREAD

**Issue by jrgriffiniii** [pulibrary/dspace-development#559]
As reported by a user:

```
All of the tasks in the task pool are currently labelled "initial review" which is incorrect for some of them. Yesterday it showed the correct labels, so we aren't sure what happened to change this. Thanks for your assistance with this issue and the Symplectic issues.
```

**Comment by jrgriffiniii** [comment]
This is closely related to, and will be resolved by the solutions introduced for #562

**Comment by jrgriffiniii** [comment]
This should be considered resolved until I receive feedback indicating otherwise.
